src/main/java/edu/cornell/kfs/fp/document/validation/impl/CuDisbursementVoucherAccountingLineTotalsValidation.java

```diff
@@ -57,12 +57,15 @@ public boolean validate(AttributedDocumentEvent event) {
                     return true;
                 }
                 // KFSMI- 5183
-                if (persistedDocument.getDocumentHeader().getWorkflowDocument().isSaved() && persistedDocument.getDisbVchrCheckTotalAmount().isZero()) {
+                if (persistedDocument.getDocumentHeader().getWorkflowDocument().isSaved() && persistedDocument
+                        .getDisbVchrCheckTotalAmount().isZero()) {
                     return true;
                 }
 
                 // check total cannot decrease
-                if (!persistedDocument.getDocumentHeader().getWorkflowDocument().isCompletionRequested() && (!persistedDocument.getDisbVchrCheckTotalAmount().equals(dvDocument.getDisbVchrCheckTotalAmount()))) {
+                if (!persistedDocument.getDocumentHeader().getWorkflowDocument()
+                        .isCompletionRequested() && persistedDocument.getDisbVchrCheckTotalAmount().isLessThan(
```

Comment:
The old cu-kfs code was using equals() instead of isLessThan() for the comparison of the totals. Can you please check to see if isLessThan() is indeed the correct method or if we purposely customized this to use equals() instead?

Reply:
It is not clear which customization changed this code but I will revert to our local customized version.